interview-questions.md

```diff
@@ -10,29 +10,29 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 Similarities
 
-1.
-2.
-3.
+1.JavaScript and Ruby are both oop languages.
```

Comment:
JavaScript has OOP-like tendencies, but as you noted in your first answer on the differences, not everything in JavaScript is an object. Technically, JavaScript uses prototypical inheritance.